.github/scripts/main/main.sh

```diff
@@ -52,6 +52,82 @@ make install
 export PATH="$PREFIX/bin:$PATH"
 opam --version
 
+if [ "$OPAM_DOC" = "1" ]; then
+  rm -rf src_ext/
+  make -C doc html man-html pages
+
+  if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
+    . .github/scripts/common/hygiene-preamble.sh
+    diff="git diff $BASE_REF_SHA..$PR_REF_SHA"
+    set +e
+    files="`$diff --name-only --diff-filter=A | grep 'src/.*mli'`"
+    set -e
+    if [ -n "$files" ]; then
+      echo '::group::new module added - checking it'
+      if $diff --name-only --exit-code -- doc/index.html ; then
+        echo '::error new module added but index not updates'
+        echo "$files"
+        exit 3
+      fi
+      echo '::endgroup::new module added - checking it'
+    else
+      echo 'No new module added'
```

Comment:
Pedantry is the order of my morning 😊
```suggestion
      echo 'No new modules added'
```